docs/native-modules-setup.md

```diff
@@ -5,22 +5,24 @@ title: Native Modules Setup
 
 Native modules are usually distributed as npm packages, except that on top of the usual Javascript they will include some native code per platform. To understand more about npm packages you may find [this guide](https://docs.npmjs.com/getting-started/publishing-npm-packages) useful.
 
-To get set up with the basic project structure for a native module we will use a third party tool [create-react-native-module](https://github.com/brodybits/create-react-native-module). You can go ahead further and dive deep into how that library works, for our needs we will only need:
+To get set up with the basic project structure for a native module we will use the community tool [@react-native-community/bob](https://github.com/react-native-community/bob)\**. You can go ahead further and dive deep into how that library works, for our needs we will only need:
 
 ```sh
-$ yarn global add create-react-native-module
-$ create-react-native-module MyLibrary
+$ npx @react-native-community/bob create react-native-awesome-module
 ```
 
-Where MyLibrary is the name you would like for the new module. After doing this you will navigate into `MyLibrary` folder and install the npm package to be locally available for your computer by doing:
+Where `react-native-awesome-module` is the name you would like for the new module. After doing this you will navigate into `react-native-awesome-module` folder and bootstrap the example project:
 
 ```sh
-$ yarn install
+$ yarn bootstrap
 ```
 
-After this is done you can go to your main react app folder (which you created by doing `npx react-native init MyApp`)
+After this is done, you'll be able to start the example app with the following command:
 
-- add your newly created module as a dependency in your package.json
-- run `yarn install` to bring it along from your local npm repository.
+```sh
+$ yarn example ios 
+// or
+$ yarn example android 
+```
 
-After this, you will be able to continue to [Native Modules (iOS)](native-modules-ios) or [Native Modules (Android)](native-modules-android) to add in some code. Make sure to read the README.md within your `MyLibrary` Directory for platform-specific instructions on how to include the project.
```

Comment:
We might want to keep this paragraph so to keep the links to native modules docs.

Reply:
Added the links back, my bad!